code/modules/clothing/suits/hoodies.dm

```diff
@@ -236,6 +236,17 @@
 	icon_state = "hoodie_sleeveless"
 	item_state = "hoodie_sleeveless"
 
+/obj/item/clothing/suit/storage/hooded/wintercoat/konyang
+	name = "konyang village coat"
+	desc = "A highly prized hooded coat with unmatched breathability and insulation. Imported from Konyang, this garment is made with a weave derived from the feathers of indigenous birds."
+	icon_state = "konyang_village"
+	item_state = "konyang_village"
+	hoodtype = /obj/item/clothing/head/winterhood/hoodie
```

Comment:
```suggestion
	hoodtype = /obj/item/clothing/head/winterhood/carp
```
Based on the description of the item below this.

Reply:
Accidentally made the carp the hoodtype.